OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp

```diff
@@ -126,6 +126,10 @@ OMSSimulationOutputWidget::OMSSimulationOutputWidget(const QString &cref, const
     QStringList args(QString("%1/share/OMSimulator/scripts/OMSimulatorServer.py").arg(Helper::OpenModelicaHome));
     args << QString("--endpoint-pub=%1").arg(QString(endPoint));
     args << QString("--model=%1").arg(fileName);
+    OMSimulatorPage *pOMSimulatorPage = OptionsDialog::instance()->getOMSimulatorPage();
+    int logLevel = pOMSimulatorPage->getLoggingLevelComboBox()->itemData(pOMSimulatorPage->getLoggingLevelComboBox()->currentIndex()).toInt();
+    args << QString("--logLevel=%1").arg(logLevel);
+    args << QString("--option=%1").arg(pOMSimulatorPage->getCommandLineOptionsTextBox()->text());
```

Comment:
Do we might get escaping issues?

Reply:
Maybe if we have any string based options but I don't see any in the Flags.cpp

Reply:
There are a few, e.g. result file, temp directory. But they are not really intended to be used together with OMEdit.

Reply:
As long as the user quotes the options we are fine.